### PR TITLE
Modify headings and image size

### PR DESCRIPTION
**Pull request by HalynaDumych**
Modify headings and image size in the **Get started** section of
the User Guide.

Relates-to: OTA-4584

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= The Workflow of an OTA Update
+= UTA update workflow
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]: OTA

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ NOTE: HERE needs to verify your date of birth to ensure that you are of the lega
 6. In *Security check*, enter the security code that you see in the picture.
 +
 [.lightbackground]
-image::img::log_in_dialog_box.png[image, align="left"]
+image::img::log_in_dialog_box.png[image,500,align="left"]
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]: Probably ok, but it's better to use percentages than pixels.

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc [hunk 3]
```diff
@@ -29,4 +29,4 @@ When you log in for the first time, to continue using the application, do the fo
 2. In the trial notification,  agree to the HERE Location Platform Services terms and conditions and privacy policy, and then click **Continue**.
 +
 [.lightbackground]
-image::img::log_in_trial.png[image]
+image::img::log_in_trial.png[image,300]
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]: Here too, better to use percentages than pixels.

File: ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc (hunk 1 above)
Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]: My bad, corrected.

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc (hunk 2 above)
Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]: Changed

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc (hunk 3 above)
Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]: Changed

**Pull request merged**